src/server/game/Maps/Map.cpp

```diff
@@ -3038,45 +3038,41 @@ bool Map::CheckRespawn(RespawnInfo* info)
 
     uint32 poolId = info->spawnId ? sPoolMgr->IsPartOfAPool(info->type, info->spawnId) : 0;
     // Next, check if there's already an instance of this object that would block the respawn
-    // Only do this for unpooled spawns
```

Comment:
this entire diff block is just un-indenting an `if`

Reply:
![image](https://user-images.githubusercontent.com/1153754/103370719-cfccf900-4acd-11eb-908a-da259654d597.png)

this setting makes the diff more readable

Reply:
nice, didn't know that

Reply:
Also easily available by just slapping `?w=1` at the end of the URL:
- https://github.com/TrinityCore/TrinityCore/pull/25785/files?w=1